_data/projects/headset-control-plus.yml

```
@@ -0,0 +1,14 @@
+name: Headset Control Plus
+desc: Unleash the power of your Headset button by supporting gesture like: Long Press, Double Press, Triple Press etc.
```

The build error is due to the second `:` here being parsed incorrectly. I recommend tweaking the words:

```suggestion
desc: Unleash the power of your Headset button by supporting gestures such as Long Press, Double Press, Triple Press etc.
```

---

Nice catch! Thank you